src/js-marketing-notification.test.ts

```diff
@@ -1,76 +1,58 @@
 import { JsMarketingNotification, Notification } from './js-marketing-notification';
 import { LocalStorageManager } from './github/gtmsportswear/js-local-storage-manager@1.0.2/local-storage-manager';
 
-let notifier: JsMarketingNotification,
-    notification: Notification,
-    lsm;
+let lsm,
+    notifier: JsMarketingNotification,
+    notification: Notification = {
+      tabContent: {
+        node: document.createElement('h2')
```

Comment:
I think we should simplify it, and do away with classNames. Sorry I know I had suggested classNames earlier probably.